axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+For example, when a Spring REST endpoint is called it will automatically start a trace.
+With the `axon-tracing-opentelemetry` module, this trace will be propagated to all subsequent Axon Framework messages.
+For example, if the REST call produces a command which is sent over Axon Server,
+handling the command will be included in the same trace as the original REST call.
+
+### Configuration
+
+To get OpenTelemetry support enabled you will need to add the following dependency to your application's dependencies:
+
+{% tabs %}
+{% tab title="Maven" %}
+
+```xml  
+<dependency>  
+    <groupId>org.axonframework</groupId>  
+    <artifactId>axon-tracing-opentelemetry</artifactId>
+	<version>${axon-framework.version}</version>
+</dependency>
+```
+
+{% endtab %}%}
+{% tab title="Gradle" %}
+
+```gradle
+implementation group: 'org.axonframework', name: 'axon-tracing-opentelemetry', version: axonFrameworkVersion
+```
+
+{% endtab %}
+{% endtabs %}
+
+Depending on your application, more configuration might be needed.
+
+## Spring boot autoconfiguration
+
+When using the Spring Boot autoconfiguration of Axon Framework, most things will be autoconfigured regardless of the
+implementation.
+
+You might want to configure certain settings that are available.
+The following table contains all configurable settings, their defaults, and what they change:
+
+| setting                                               | Default | Description                                                                                            |
+|-------------------------------------------------------|---------|--------------------------------------------------------------------------------------------------------|
+| `axon.tracing.showEventSourcingHandlers`              | `false` | Whether to show event sourcing handlers as a trace. This can be very noisy and is disabled by default. |
+| `axon.tracing.attributeProviders.aggregateIdentifier` | `true`  | Whether to add the aggregate identifier as a label when handling a message                             |
+| `axon.tracing.attributeProviders.messageId`           | `true`  | Whether to add the message identifier as a label when handling a message                               |
+| `axon.tracing.attributeProviders.messageName`         | `true`  | Whether to add the message name as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.messageType`         | `true`  | Whether to add the message type as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.payloadType`         | `true`  | Whether to add the payload type as a label when handling a message                                     |
+| `axon.tracing.attributeProviders.metadata`            | `true`  | Whether to add the metadata properties as labels when handling a message                               |
+
+#### Manual configuration
```

Comment:
Ah, here it is. I'd expect this to be part of a code tab I guess. Any how, not after all the Spring Boot configuration stuff.

Reply:
I'm not sure what you mean. It's in the auto-configuration section of OpenTelemetry.  Where else would you like it?